src/Illuminate/Testing/Fluent/AssertableJson.php

```diff
@@ -79,14 +79,14 @@ protected function prop(string $key = null)
      * @param  \Closure  $callback
      * @return $this
      */
-    protected function scope(string $key, Closure $callback): self
+    protected function scope(string $key, Closure $callback): static
```

Comment:
You can't use the `static` return type here. Leave it as `self. Return types are already contravariant, so should be fine.

Reply:
Thank you! It is indeed fine without changing the return type.